### PR TITLE
Test that npm packages has private false

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Hi @tbergq, sorry for the delays. I wanted to check one thing more closely, but I didn't get to it yet.

**Comment by tbergq** [comment]
No worries, take your time. Note that packages are now alphabetically sorted 🤓

**Comment by tbergq** [comment]
This script will also fail if we try to release a package that does not exist. Which is how I found #103

File: src/monorepo-utils/src/Workspaces.js [hunk 1]
```diff
@@ -69,4 +70,17 @@ module.exports = {
     });
     return packageJSONLocations;
   },
+
+  getWorkspaceLocationMap(): $ReadOnlyMap<string, string> {
```
Comment (mrtnzlml) [inline src/monorepo-utils/src/Workspaces.js]: @tbergq Why isn't the `getWorkspacesSync` good enough for your use-case? The functions which are already available here seems to be covering your case as well. 🤔

File: scripts/publishedPackages.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[
```
Comment (mrtnzlml) [inline scripts/publishedPackages.json]: TODO before merging: rebase this PR and update this list.

File: src/monorepo-utils/src/Workspaces.js (hunk 1 above)
Comment (tbergq) [reply src/monorepo-utils/src/Workspaces.js]: `getWorkspacesSync ` only gives me the path to each package.json file. I would have to read every json file and check the name.

I am all for using existing functionality, I just can't find anything that fits the use case well

Comment (tbergq) [reply src/monorepo-utils/src/Workspaces.js]: @mrtnzlml please check again

File: src/monorepo-scanner/src/scans/NpmPackages.scan.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  workspaceMap.set(packageJson.name, packageJson);
+}
+
+// $FlowAllowDynamicImport
+const npmPackages = require(path.join(root, 'scripts', 'publishedPackages.json'));
+
+for (const npmPackage of npmPackages) {
+  test(`${npmPackage} has private false flag`, () => {
+    const packageJson = workspaceMap.get(npmPackage);
+    expect(packageJson != null).toGiveHelp('You are trying to release a non existing package');
```
Comment (jaroslav-kubicek) [inline src/monorepo-scanner/src/scans/NpmPackages.scan.js]: `!=` instead `!==` is intentional?

Comment (tbergq) [reply src/monorepo-scanner/src/scans/NpmPackages.scan.js]: Probably it will only be undefined here, so I could change that yes

Comment (tbergq) [reply src/monorepo-scanner/src/scans/NpmPackages.scan.js]: 🚀

File: src/monorepo-scanner/src/scans/NpmPackages.scan.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+// $FlowAllowDynamicImport
+const npmPackages = require(path.join(root, 'scripts', 'publishedPackages.json'));
+
+for (const npmPackage of npmPackages) {
+  test(`${npmPackage} has private false flag`, () => {
+    const packageJson = workspaceMap.get(npmPackage);
+    expect(packageJson !== undefined).toGiveHelp(
+      'You are trying to release a non existing package',
```
Comment (mrtnzlml) [inline src/monorepo-scanner/src/scans/NpmPackages.scan.js]: Yaay! 👍

**Pull request merged**